### ISSUE TITLE
Progress?

### ISSUE THREAD

**Issue by ripSquid** [nikita600/picoCart_NES#1]
Progress?

**Comment by nikita600** [comment]
Oh shi~
I don't even notice that repo in public.

Unfortunately, progress stuck on 2021, when I built protoype PCB and starts writing code, then I have a lot of work on my main job, so I just lost motivation to continue project, but I still want to finish this project.

Anyway, here some old photos of proto PCB:
![FrontPCB](https://user-images.githubusercontent.com/510482/235535780-51ea5008-1e25-463a-9b26-c8da0ed797f0.jpg)
![BackPCB](https://user-images.githubusercontent.com/510482/235535775-8db7d591-9f35-4a7f-b210-c70845962e81.jpg)

**Comment by CodeAsm** [comment]
Ow man that looks like cool progress nonetheless :smile: 
are you also using a cpld? or what is that big ic there? is that also providing save voltage conversion to and from the NES bus?

If not and maybe never really ready to publish code, you could use this repo to atleast document what youve made sofar as a blog like thing? ive been told creating a portfolio could help in the job market, so i myself try to work on my own

**Comment by nikita600** [comment]
> Ow man that looks like cool progress nonetheless 😄 are you also using a cpld? or what is that big ic there? is that also providing save voltage conversion to and from the NES bus?
> 
> If not and maybe never really ready to publish code, you could use this repo to atleast document what youve made sofar as a blog like thing? ive been told creating a portfolio could help in the job market, so i myself try to work on my own

Yup, that's CPLD XC94144XL. Yes, it's initially works as voltage level shifter, but it should pack NES cartridge slot state every cycle and send this data to RP2040 via SPI. But still it's only a theory...

Anyway, I still hope, that I can make another proto PCB and starts to upload some code or something :D
